appveyor.yml

```
@@ -63,7 +65,8 @@ test_script:
     .\codecov.exe --dir "./.coverage/"
 
 after_test:
-- dotnet pack PocketCsvReader -p:version="%GitVersion_SemVer%" -c Release --include-symbols --no-build --nologo 
+- dotnet pack PocketCsvReader -p:version="%GitVersion_SemVer%" -c Release --include-symbols --no-build --nologo
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)